test/data/handlers/fixtures/response/schedulers_handler/create_scheduler.json

```diff
@@ -76,6 +76,10 @@
           "metadata": {}
         }
       }
-    ]
+    ],
+    "annotations": [{
+      "name": "annotation",
```

Comment:
this looks cumbersome, what about using the annotations as a map like kubernetes, it would look better IMO

```
"annotations": {
     "http://enable-something": "true",
    "http://enable-something-else": "true"
}
```

Reply:
I also do prefer that could be this way but we need to have mapped annotations, right?

Reply:
yes, you have to change your mapping to use this format

Reply:
I tried to follow the same interpretation from the environment. The YAML is stored as a map but the API contract I think is best to be this way.